src/components/sbb-checkbox/sbb-checkbox.scss

```
@@ -0,0 +1,117 @@
+@include host-component-properties;
+
+:host {
+  --checkbox-border-width: var(--sbb-border-width-1x);
+  --checkbox-width: var(--sbb-size-icon-ui-small);
+  --checkbox-height: var(--checkbox-width);
+  --checkbox-border-color: var(--sbb-color-smoke-default);
+  --checkbox-selection-color: var(--sbb-color-red-default);
+  --checkbox-label-color: var(--sbb-color-charcoal-default);
+  --checkbox-cursor: pointer;
+  --checkbox-border-radius: #{px-to-rem-build(4)};
+
+  display: inline-block;
+}
+
+:host([disabled]) {
+  --checkbox-color: var(--sbb-color-graphite-default);
+  --checkbox-border-color: var(--sbb-color-graphite-default);
+  --checkbox-selection-color: var(--sbb-color-graphite-default);
+  --checkbox-label-color: var(--sbb-color-smoke-default);
+  --checkbox-cursor: not-allowed;
+}
+
+.sbb-checkbox {
+  @include text-s--regular;
+
+  position: relative;
+  display: block;
+  color: var(--checkbox-label-color);
+  cursor: var(--checkbox-cursor);
+
+  :host([size='m']) & {
+    @include text-m--regular;
+  }
+}
+
+.sbb-checkbox__inner {
+  display: flex;
+  align-items: start;
+  gap: var(--sbb-spacing-fixed-2x);
+}
+
+input[type='checkbox'] {
+  @include hiddenCheckbox;
+}
+
+.sbb-checkbox__aligner,
+.sbb-checkbox__label--icon {
+  display: flex;
+  align-items: center;
+  height: calc(var(--sbb-font-size-text-s) * var(--sbb-typo-line-height-body-text));
+
+  :host([size='m']) & {
+    height: calc(var(--sbb-font-size-text-m) * var(--sbb-typo-line-height-body-text));
+  }
+}
+
+.sbb-checkbox__selection {
+  position: relative;
+  min-width: var(--checkbox-width);
+  height: var(--checkbox-height);
+  border-radius: var(--checkbox-border-radius);
+  border: var(--checkbox-border-width) solid var(--checkbox-border-color);
+  background-color: var(--sbb-color-white-default);
+}
+
+// todo: icon is bigger then --checkbox-width
+.sbb-checkbox__icon {
+  display: inline-flex;
+  position: absolute;
+
+  // We stretch the icon container over the selection element, in order to allow
+  // full width of the svg icon without ignoring the selection border.
+  margin: calc(var(--sbb-border-width-1x) * -1);
+  width: var(--checkbox-width);
+  height: var(--checkbox-height);
+
+  svg {
+    margin: auto;
+  }
+
+  path {
+    stroke: var(--checkbox-selection-color);
+  }
+}
+
+.sbb-checkbox__label {
+  display: flex;
+  gap: var(--sbb-spacing-fixed-2x);
+  color: var(--checkbox-label-color);
+
+  // fix for Chrome and Safari line-height approximation
+  line-height: max(
+    (var(--sbb-font-size-text-s) * var(--sbb-typo-line-height-body-text)),
```

Could maybe simplified to this:
```suggestion
    (1em * var(--sbb-typo-line-height-body-text)),
```